code/modules/jobs/job_types/cook.dm

```diff
@@ -18,6 +18,17 @@
 	display_order = JOB_DISPLAY_ORDER_COOK
 	bounty_types = CIV_JOB_CHEF
 
+	mail_goodies = list(
+		/obj/item/storage/box/ingredients/random = 80,
+		/datum/reagent/consumable/caramel =  20,
+		/obj/item/reagent_containers/food/condiment/flour = 20,
+		/obj/item/reagent_containers/food/condiment/rice = 20,
+		/obj/item/reagent_containers/food/condiment/enzyme = 15,
+		/obj/item/reagent_containers/food/condiment/soymilk = 15,
+		/obj/item/kitchen/knife = 4,
+		/obj/item/kitchen/knife/butcher = 2
```

Comment:
these are so low chance but are pretty much useless for a job that can vend them